contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/MarkdownEditor/MarkdownEditor.vue

```diff
@@ -576,13 +583,16 @@
           selection.setEnd(selection.endContainer.nextSibling, 1);
           squire.setSelection(selection);
         }
+        // Important debugging tip... If the editor selection is broken,
+        // uncomment the following line to understand how it got that way:
+        // console.log("keypress", selection, event);
```

Comment:
This is like the only effective way I found to actually debug the TUI/Squire cursor selection issues, so I think it's worth having in here.